docs/detections/prebuilt-rules/prebuilt-rules-changelog.asciidoc

```diff
@@ -5,6 +5,8 @@ The following lists prebuilt rule updates per release. Only rules with
 significant modifications to their query or scope are listed. For detailed
 information about a rule's changes, see the rule's description page.
 
+To see the updates for 8.7 and further releases, please visit the https://www.elastic.co/guide/en/security/8.7/prebuilt-rules-downloadable-updates.html[last version] for the updates.
```

Comment:
@approksiu @terrancedejesus I think we need to remove this page altogether right?

Reply:
We need to keep this page (and update it to link to the downloadable rule updates per https://user-images.githubusercontent.com/1636709/256029653-0df94b78-fefb-4f86-afea-b4c4f81ea903.png). It is still referenced from security solution on 8.8.